test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -0,0 +1,38 @@
+# Owner(s): ["module: onnx"]
```

Comment:
Should we rename to test_fx_passes.py?

Reply:
I suggested the name. We can discussion more if you have suggestions, thanks!

Reply:
Reproducing from the chat: since this is testing from export end to end, I suggested we should include the test here. Otherwise for it to be a unit test we should probably be calling the pass directly on an fx graph.